runtime/sema/check_composite_declaration.go

```diff
@@ -1017,23 +1022,39 @@ func (checker *Checker) checkCompositeConformance(
 		if interfaceMember.Predeclared {
 			return
 		}
-
 		compositeMember, ok := compositeType.Members.Get(name)
 		if !ok {
 			if options.checkMissingMembers {
-				missingMembers = append(missingMembers, interfaceMember)
+				if interfaceMember.DeclarationKind != common.DeclarationKindFunction {
+					missingMembers = append(missingMembers, interfaceMember)
+				} else {
+					if !interfaceMember.HasImplementation {
+						missingMembers = append(missingMembers, interfaceMember)
+					} else {
+						if (*overridden)[name] {
+							checker.report(
+								&MultipleInterfaceDefaultImplementationsError{
+									CompositeType: compositeType,
+									Member:        interfaceMember,
+								},
+							)
+						}
+						(*overridden)[name] = true
+					}
+
+				}
 			}
-			return
 		}
 
-		if !checker.memberSatisfied(compositeMember, interfaceMember) {
+		if compositeMember != nil && !checker.memberSatisfied(compositeMember, interfaceMember) {
```

Comment:
here covers the case where compositeMember is inherited from interface. so now it can be missing (nil) but can cause no error in !ok section (if interface implements)